services/web/server/src/simcore_service_webserver/projects/_metadata_api.py

```diff
@@ -0,0 +1,30 @@
+from aiohttp import web
+from models_library.api_schemas_webserver.projects_metadata import MetadataDict
+from models_library.projects import ProjectID
+from models_library.users import UserID
+
+from ..db.plugin import get_database_engine
+from ._access_rights_api import validate_project_ownership
+from ._metadata_db import get_project_metadata, upsert_project_metadata
+
+
+async def get_project_custom_metadata(
```

Comment:
I see that sometimes you call it metadata, and sometimes custom_metadata. I think I would just go for metadata to keep it simple. If you need to change it later we can always migrate the column...